recipes-bsp/dt-overlay-at91/dt-overlay-at91_git.bb

```diff
@@ -24,6 +24,12 @@ AT91BOOTSTRAP_MACHINE ??= "${MACHINE}"
 do_compile[depends] += "virtual/kernel:do_deploy virtual/kernel:do_shared_workdir"
 do_compile[nostamp] = "1"
 
+do_configure_append() {
+    if ${@bb.utils.contains('INITRAMFS_IMAGE_BUNDLE','1','true','false',d)}; then
+        sed -i -e "s#\"./zImage\"#\"./zImage-initramfs-${MACHINE}.bin\"#" ${S}/${AT91BOOTSTRAP_MACHINE}.its
```

Comment:
Hi,
Nit: isn't it possible to use python built-in capabilities to do the same?

Reply:
@noglitch I am not sure if I got your question, here we need to evaluate a bitbake variable and edit the source code based on result. full python capabilities are used e.g for assigning values to variables but not when operation require there evaluations. secondly,  do_configure is a shell function, having a function/task fully written in python may give opportunities to write it in python too

Reply:
ok, got it. Thanks for your input @kraj !